scripts/scripts.js

```diff
@@ -428,6 +438,7 @@ export function decorateMain(main) {
   decorateH3(main);
   decorateLinkedPictures(main);
   decorateLinks(main);
+  decorateAuthorBlock(main);
```

Comment:
The template specific decorations should be done in the `buildAutoBlocks` function.

We normally create these kind of decorate functions for generic things that are applicable on multiple pages, potentially the whole website.

Reply:
I remember, fixing this delimiter issues. Let me look into the code again.